alerts/cluster-monitoring-operator/PrometheusDuplicateTimestamps.md

```diff
@@ -0,0 +1,64 @@
+# PrometheusDuplicateTimestamps
+
+## Meaning
+
+The `PrometheusDuplicateTimestamps` alert is triggered when there is a constant increase in dropped samples
+due to them having identical timestamps.
+
+## Impact
+
+Unwanted samples might be scraped, and desired samples might be dropped. Consequently, queries may yield
+incorrect and unexpected results.
+
+## Diagnosis
+
+1. Determine whether the alert has triggered for the instance of Prometheus used
+   for default cluster monitoring or for the instance that monitors user-defined
+   projects by viewing the alert message's `namespace` label: the namespace for
+   default cluster monitoring is `openshift-monitoring`; the namespace for user
+   workload monitoring is `openshift-user-workload-monitoring`.
+
+2. Review the logs for the affected Prometheus instance:
+
+   ```shell
+   $ NAMESPACE='<value of namespace label from alert>'
+
+   $ oc -n $NAMESPACE logs -l 'app=prometheus' | \
+   grep 'Error on ingesting samples with different value but same timestamp.*' | sort | uniq -c | sort -n
+   level=warn ... scrape_pool="the-scrape-pool" target="an-involved-target" msg="Error on ingesting samples with different value but same timestamp"
+   ```
+
+   The logs should help identify the involved target(s).
+
+## Mitigation
```

Comment:
I would split the mitigation between platform and user-defined monitoring:
* In the first case, the answer is "open a support case".
* In the second case, the admin needs to find the guilty Service/Pod Monitor